reps/config.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  rootPath: "../../../../mozilla.org"
```

Comment:
We might want to rather change this to:

`"../../../../mozilla.org/fx-team"`

and rename the ``rootPath` to `firefoxSrc` or something like that, what do you think?

Honza